pkg/netpol/diff/diff_formatter.go

```diff
@@ -54,16 +58,33 @@ func formDiffFieldsDataOfChangedConns(changedConns []*ConnsPair) (netpolsChanged
 	return netpolsRes, ingressRes
 }
 
-func formDiffFieldsDataOfRemovedConns(removedConns []connlist.Peer2PeerConnection) (netpolsRemoved, ingressRemoved []*singleDiffFields) {
+func formDiffFieldsDataOfRemovedConns(removedConns []RemovedConnsPeers) (netpolsRemoved, ingressRemoved []*singleDiffFields) {
```

Comment:
can avoid code duplication for `formDiffFieldsDataOfRemovedConns` and `formDiffFieldsDataOfAddedConns` ?